requirements.tox.txt

```diff
@@ -3,3 +3,5 @@ py==1.5.4
 six==1.11.0
 tox==3.2.1
 virtualenv==16.0.0
+xlwt==1.3.0
+xlrd==1.1.0
```

Comment:
These aren't required to install `tox` itself.  Only in the requirements files above which tox uses to install in the envs it creates.  And yes, this is a mess.  I don't like how it is (my fault) nor do I have an overall good solution to it presently.